modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/exception/mapper/PrincipalExceptionMapper.java

```diff
@@ -44,7 +44,7 @@ protected Problem getProblem(PrincipalException principalException) {
 			status = Response.Status.NOT_FOUND;
 		}
 
-		return new Problem(status, null);
+		return new Problem(status, principalException.getMessage());
```

Comment:
Don't know if we were avoiding to return the exception message for some reason but I've felt that it was better this way to give more information to the API Consumer